image/image.go

```diff
@@ -24,9 +25,10 @@ var (
 // may not be a `genericImage` directly. However, most users of `types.Image`
 // do not care, and those who care about `skopeo/docker.Image` know they do.
 type genericImage struct {
-	src              types.ImageSource
-	cachedManifest   []byte   // Private cache for Manifest(); nil if not yet known.
-	cachedSignatures [][]byte // Private cache for Signatures(); nil if not yet known.
+	src                    types.ImageSource
+	cachedManifest         []byte   // Private cache for Manifest(); nil if not yet known.
+	cachedManifestMIMEType string   // Private cache for the manifest media type w/o having to guess it
```

Comment:
Could this comment also document:
- valid only if `cachedManifest` is not `nil
- may be `""` if unknown.
?